lib/vsc/mympirun/rm/pbs.py

```diff
@@ -37,6 +37,8 @@ class PBS(Sched):
     SCHED_ENVIRON_ID = 'PBS_JOBID'
     SCHED_ENVIRON_NODEFILE = 'PBS_NODEFILE'
 
+    # pbsssh is only used if native support for pbsdsh is not supported in the MPI library;
+    # e.g. Intel MPI v5.x and newer supports using pbsdsh as launcher natively, no need for pbsssh wrapper
```

Comment:
for OpenMPI we shouldn't use `pbsssh` either, OpenMPI will use a correct launcher mechanism by default if it was built with Torque integration, see https://www.open-mpi.org/faq/?category=tm